### PR TITLE
Adjust spacing in csproj files

### PR DESCRIPTION
**Pull request by jbagga**
Addresses #462

### CODE REVIEW

File: src/Microsoft.DotNet.Web.ProjectTemplates/RazorPagesWeb-CSharp.csproj.in [hunk 1]
```diff
@@ -8,7 +8,9 @@
     <WebProject_DirectoryAccessLevelKey Condition="'$(OrganizationalAuth)' == 'True' AND '$(OrgReadAccess)' != 'True'">0</WebProject_DirectoryAccessLevelKey>
     <WebProject_DirectoryAccessLevelKey Condition="'$(OrganizationalAuth)' == 'True' AND '$(OrgReadAccess)' == 'True'">1</WebProject_DirectoryAccessLevelKey>
   </PropertyGroup>
+  <!--#if (IndividualLocalAuth && UseLocalDB) -->
```
Comment (fredrik-lundin) [inline src/Microsoft.DotNet.Web.ProjectTemplates/RazorPagesWeb-CSharp.csproj.in]: What is this empty region for?

Comment (mkArtakMSFT) [reply src/Microsoft.DotNet.Web.ProjectTemplates/RazorPagesWeb-CSharp.csproj.in]: Why not have an empty line always in between `PropertyGroup` and `ItemGroup` here?

Comment (jbagga) [reply src/Microsoft.DotNet.Web.ProjectTemplates/RazorPagesWeb-CSharp.csproj.in]: The `ItemGroup` right below only applies for certain conditions - using IndividualLocalAuth and if UseLocalDB is false. So when we add the ItemGroup we add the space.

**Comment by jbagga** [comment]
The UseLocalDB condition was incorrect for the space in MVC. Corrected for it

File: build/repo.props [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <!-- These properties are use by the automation that updates dependencies.props -->
     <LineupPackageId>Internal.AspNetCore.Universe.Lineup</LineupPackageId>
-    <LineupPackageVersion>2.1.0-rc1-*</LineupPackageVersion>
+    <LineupPackageVersion>2.1.0-rtm-*</LineupPackageVersion>
```
Comment (jbagga) [inline build/repo.props]: @ryanbrandenburg FYI. Updating this for templating to fix the build

**Pull request merged**